package.json

```diff
@@ -19,6 +19,6 @@
     "coffeelint": "^1.10.1"
   },
   "dependencies": {
-    "tree-sitter-json": "^0.13.1"
+    "tree-sitter-json": "git://github.com/tree-sitter/tree-sitter-json.git#v0.14.0"
```

Comment:
We can change this to `^0.14.0` once that version has been published to npm. Until then, in order to push this work forward, we can depend on the [v0.14.0 tag](https://github.com/tree-sitter/tree-sitter-json/releases/tag/v0.14.0).